### DISCUSSION TITLE
Integerate an existing yaml file in a .md document

### DISCUSSION THREAD

**Discussion by revit13** [mkdocs/mkdocs#2539] [Q&A]
I am looking for a way to integrate an existing yaml file in a .md document. the yaml file resides in a gitub repo and can also be accessed relative to the .md file (for example ../dir/test.yaml)
I wonder if there is a way to do so in mkdocs. Thanks

**Comment by mondeja** [comment]
You can use [mkdocs-include-markdown-plugin](https://github.com/mondeja/mkdocs-include-markdown-plugin) with the [include directive](https://github.com/mondeja/mkdocs-include-markdown-plugin#include).
